compiler/z/codegen/S390Instruction.cpp

```diff
@@ -5377,7 +5377,7 @@ TR::S390VirtualGuardNOPInstruction::generateBinaryEncoding()
 #ifdef DEBUG
       if (debug("traceVGNOP"))
          {
-         printf("####> virtual location(est) = %p+%d, label (relocation) = %p\n", cursor, distance, label);
+         printf("####> virtual location(est) = %p+%ld, label (relocation) = %p\n", cursor, distance, label);
```

Comment:
Although we're not consistent (yet) throughout the codebase, the preference is to use the `OMR_PRIxPTR` or `OMR_PRIdPTR` format specifiers for `intptr_t` types.